### PR TITLE
Feature/13604 make validate content builds script pass

### PR DESCRIPTION
**Pull request by timwright12**
This validation script compares the output of vets-website to that of content-build to make sure we didn't miss anything

### CODE REVIEW

File: jenkins/common.groovy [hunk 1]
```diff
@@ -265,7 +266,7 @@ def buildAll(String ref, dockerContainer, Boolean contentOnlyBuild) {
 
 def prearchive(dockerContainer, envName) {
   dockerContainer.inside(DOCKER_ARGS) {
-    sh "cd /application && node --max-old-space-size=8192 script/prearchive.js --buildtype=${envName}"
+    sh "cd /application && node --max-old-space-size=16384 script/prearchive.js --buildtype=${envName}"
```
Comment (timwright12) [inline jenkins/common.groovy:269]: Temp fix for our JS heap error

File: Jenkinsfile [hunk 2]
```diff
@@ -62,32 +62,6 @@ node('vetsgov-general-purpose') {
     }
   }
 
-  // // Run E2E and accessibility tests
```
Comment (timwright12) [inline Jenkinsfile:65]: Just cleaning these up, they run on a schedule, not in CI

File: Jenkinsfile.content [hunk 3]
```diff
@@ -62,7 +62,7 @@ node('vetsgov-general-purpose') {
   dockerContainer = commonStages.setup()
 
   stage("Build") {
-    commonStages.build(ref, dockerContainer, ref, params.env, false, true)
+    commonStages.build(ref, dockerContainer, ref, params.env, false, true, '/application')
```
Comment (timwright12) [inline Jenkinsfile.content:65]: I started implementing the CI solution before noticing that is was actually a different ticket, but just this in

File: script/content-build-compare.js [hunk 4]
```diff
@@ -72,17 +73,15 @@ function compareBuilds(buildtype) {
     'websiteContentBuildHash.txt',
   );
   const standaloneContentBuild = hashBuildOutput(
-    path.join(__dirname, `../../content-build/build/${buildtype}`),
```
Comment (U-DON) [inline script/content-build-compare.js:75]: Should the names of these values and filenames be swapped (`websiteContentBuild` <-> `standaloneContentBuild`)?

**Review by U-DON — APPROVED** [review]

File: script/content-build-compare.js (hunk 4 above)
Comment (timwright12) [reply script/content-build-compare.js:75]: I think I'm actually going to completely change them, they were just in there in the original PR

**Pull request merged**